test.py

```diff
@@ -45,11 +51,11 @@ def parse_options(self):
         (self.options,self.args) = parser.parse_args()
 
     def run(self,info='names',surname='A',given='Maria',
-            birth=1857,birth_start=1850,birth_end=1860,
-            death=1857,death_start=1850,death_end=1860,
-            marriage=1857,marriage_start=1850,marriage_end=1860,
-            family_surname='Nicotra',family_given='Maria',
-            criteria='surname=N:birthrange=1820-1840:deathrange=1900-1910'):
+            birth=1968,birth_start=1940,birth_end=1960,
```

Comment:
@MatthiasJobst These changes seem to be opinionated and outside the scope of upgrading to a newer version of Python. A quick, cursory review of the other changes did not stand out as out of scope to me.

CC: @zappala